modules/ROOT/pages/apikit-odatav4-expand-feature.adoc

```diff
@@ -45,10 +50,12 @@ You can configure how the software handles errors during the expansion operation
 
 The `generic expand` feature dispatches additional queries to the application when the original flow completes. Because this is done based on the application `EDM`, you must make extra annotations to add enough context to the module:
 
-. The `EntitySet` where the expansion takes place has a `NavigationPropertyBinding` for every expandable `NavigationProperty` of its `Entity`. +
+* The `EntitySet` where the expansion takes place has a `NavigationPropertyBinding` for every expandable `NavigationProperty` of its `Entity`. 
++
 This is used to select which `<entity-collection-source/>` is executed when expanding that property.
-. Each expandable `NavigationProperty` has a `ReferentialConstraint` attached. +
-Internal requests use this, when the value of `Property` is matched against the value of `ReferencedProperty`.
+* Each expandable `NavigationProperty` has a `ReferentialConstraint` attached. 
++
+Internal requests use it when the value of `Property` is matched against the value of `ReferencedProperty`.
```

Comment:
```suggestion
Internal requests use `ReferentialConstraint` when matching the value of `Property` against the value of `ReferencedProperty`.
```
It's a bit unclear what "it" refers to in this case. Please confirm if the suggestion is correct.

Reply:
Yes! You're right! Thanks!